### ISSUE TITLE
loss info

### ISSUE THREAD

**Issue by DecentMakeover** [LiuHao-THU/3D-Segmentation-Pytorch-version#1]
Hi 

Thanks for sharing your work , i wanted to know what other loss functions do you recommend for 3dsegmentation  tasks? 

and what is the improvement of dice over other loss functions, currently i am using cross entropy.

Thanks
